src/EnergyPlus/MixedAir.cc

```diff
@@ -1415,6 +1415,16 @@ namespace MixedAir {
 		NumERVControllers = GetNumObjectsFound( CurrentModuleObjects( CMO_ERVController ) );
 		NumOAControllers += NumERVControllers;
 
+
+		// Find number of primary air systems - fixes 5476
+		NumPrimaryAirSys = GetNumObjectsFound( "AirLoopHVAC" );
+		int NumOACntrlOrPrimAirSys = max( NumPrimaryAirSys, NumOAControllers );
+		AirLoopControlInfo.allocate( NumOACntrlOrPrimAirSys );
```

Comment:
Why allocate here to NumOAControlOrPrimAirSys?  Seems wrong to set up these full airloop arrays just for ERV OA controllers.